cfme/tests/services/test_orchestration_template.py

```diff
@@ -39,6 +40,9 @@
 """
 
 
+METHOD_TORSO_empty = ""
```

Comment:
@niyazRedhat hehe, you mentioned `@milan` and not me. Well, for an empty string, you don't have to have some specific constant somewhere, just use an empty string where you want to use it.

btw. constants' names must be all uppercase.